subsys/settings/src/settings_store.c

```diff
@@ -158,6 +158,21 @@ int settings_save(void)
 	return rc;
 }
 
+int settings_storage_get(void **storage)
+{
+	struct settings_store *cs = settings_save_dst;
+
+	if (!cs) {
+		return -4;
```

Comment:
defined errno code for magic number?

Reply:
that remains after debugging :D - will remove.